includes/wikia/parser/templatetypes/handlers/NavboxTemplate.class.php

```diff
@@ -1,12 +1,52 @@
 <?php
 
 class NavboxTemplate {
+
+	const MARK = 'NAVBOXUNIQ';
+
+	public static function handle( $text ) {
+		return !empty( $text ) ? self::mark( $text ) : $text;
+	}
+
+	public static function resolve( &$html ) {
+		if ( $html ) {
+			$html = self::process( $html );
+		}
+		return true;
+	}
+
+	private static function process( $html ) {
```

Comment:
I understand why this was not abstracted away together with https://github.com/Wikia/app/pull/10421/files#diff-4ef504b27eb589fdde574f4f6e9677d3R31, though do you think it makes sense to add a tracking TODO just in case we decide to leave this part of code after the A/B test?